### PR TITLE
DEM-1103 Add confirmation modal logic & Email mock data

### PR DESCRIPTION
**Pull request by samsep**
ping @appirio-tech/demandside

### CODE REVIEW

File: src/scripts/controllers/type.controller.coffee [hunk 1]
```diff
@@ -56,6 +56,14 @@ SubmitWorkTypeController = ($scope, $rootScope, Optimist, SubmitWorkService) ->
       SubmitWorkService.save(updates).then ->
         vm.showSuccessModal = true
 
+  vm.saveEmail = ->
+    email = vm.work.email
+    if email
```
Comment (vietnogi) [inline src/scripts/controllers/type.controller.coffee]: code styles

File: src/scripts/controllers/type.controller.coffee [hunk 2]
```diff
@@ -56,6 +56,14 @@ SubmitWorkTypeController = ($scope, $rootScope, Optimist, SubmitWorkService) ->
...
+      update =
+        email:email
```
Comment (vietnogi) [inline src/scripts/controllers/type.controller.coffee]: code styles

File: src/scripts/controllers/type.controller.coffee [hunk 3]
```diff
@@ -56,6 +56,14 @@ SubmitWorkTypeController = ($scope, $rootScope, Optimist, SubmitWorkService) ->
...
+      SubmitWorkService.save(update).then ->
+        $state.go('submit-work-features')
```
Comment (vietnogi) [inline src/scripts/controllers/type.controller.coffee]: code styles

**Comment by aselbie** [comment]
@samsep You should check in the the backend team about how the whole email/account creation/project assignment thing is going to work. I would guess that the email address probably won't be stored on the work object.

**Comment by samsep** [comment]
@aselbie we won't be showing the modal anymore since for now the user will always be logged in when creating a project. It will be added later on once we have a better understanding of how that flow will work.

**Pull request merged**